tests/unittests/Foundation/NSJSONSerializationTests.m

```diff
@@ -124,13 +124,13 @@ void VerifyDataWithJSONObjectThrows(id testObject) {
     id testObject6 = @{ @"foo" : @"1", @"bar" : @"2" };
     id testObject7 = @{ @"foo" : @{ @"bar" : @"1" }, @"foo2" : @{ @"bar2" : @"2" } };
     id testObject8 = [NSUUID UUID];
-    id testObject9 = @{ @"key" : @"value", @"nullkey" : [NSNull null], @"array" : @[ @{ @"subnullkey" : [NSNull null] } ] };
```

Comment:
We need NSNull to work correctly as a key here.  Lets due the diligence here..

---
In reply to: [80960041](https://github.com/Microsoft/WinObjC/pull/1043#discussion_r80960041) [](ancestors = 80960041,80824620)